nautobot/docs/requirements.txt

```
@@ -1,2 +1,3 @@
-mkdocs==1.2.3
+Jinja2==3.0.3
+mkdocs==1.3.0
```

```suggestion
Jinja2==2.11.3
mkdocs==1.3.0
```

Is this your suggestion @glennmatthews?

---

Yes, for `develop` at this time.